### ISSUE TITLE
brew packaging for Linux, please

### ISSUE THREAD

**Issue by silopolis** [contour-terminal/contour#1342]
It'd be a very nice to have alternative as more and more projects offer this option, especially in the Dev and Ops fields.

Flatpack may be nice for a more Desktopy use case, and fighting with deb/rpm packaging for IT pro workstations when you have options like Homebrew start to seem like a waste of resources, even for an old beard like me.

**Comment by Yaraslaut** [comment]
Hi @silopolis , thanks for the interest in `contour` we are planning to release 0.4 soon,  at the moment @christianparpart  fixing packaging for macos and once this is done we can look into packaging for brew for both platforms

**Comment by christianparpart** [comment]
Actually, brew is not maintained by us. Somebody is kind enough to keep it updated on brew. But I am positive that we can get it there for Linux as well (as soon as 0.4.0 is out of the house)
